internal/stacks/stack_manager.go

```diff
@@ -1049,18 +1090,54 @@ func (s *StackManager) CreateAccount(args []string) (string, error) {
 }
 
 func (s *StackManager) getBlockchainProvider(verbose bool) blockchain.IBlockchainProvider {
+
+	if s.Stack.BlockchainProvider == types.GoEthereum.String() {
+		s.Stack.BlockchainProvider = types.Ethereum.String()
+		s.Stack.BlockchainNodeProvider = types.GoEthereum.String()
+	}
+
+	if s.Stack.BlockchainProvider == types.HyperledgerBesu.String() {
+		s.Stack.BlockchainProvider = types.Ethereum.String()
+		s.Stack.BlockchainNodeProvider = types.HyperledgerBesu.String()
+	}
+
+	s.Stack.DisableTokenFactories = true
```

Comment:
Why are we disabling token factories here?

Reply:
It's the default. Token factories are only enabled on two paths.